src/dbus/server/dbus_thread_object.cpp

```diff
@@ -27,6 +27,7 @@
  */
 
 #include <assert.h>
+#include <fstream>
```

Comment:
nit: include C++ library headers in separate block above C library headers.

Reply:
this is not needed any more.
```suggestion
```